types/types.go

```diff
@@ -102,8 +102,8 @@ const (
 	LonghornLabelOrphan                   = "orphan"
 	LonghornLabelOrphanType               = "orphan-type"
 	LonghornLabelCRDAPIVersion            = "crd-api-version"
-
-	LonghornLabelValueEnabled = "enabled"
+	LonghornLabelVolumeAccessMode         = "volume-access-mode"
+	LonghornLabelValueEnabled             = "enabled"
```

Comment:
Keep `LonghornLabelValueEnabled` separate from the label keys.

Reply:
Got it, I will leave a blank line above the `LonghornLabelValueEnabled`.

https://github.com/longhorn/longhorn-manager/blob/31963edf0b2c3c39aee5840e4e2a0d778cb893f4/types/types.go#L105-L107